### PR TITLE
Update news and spec version - v1.3.x

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6136/ for details (Mellanox internal link).

File: NEWS [hunk 1]
```diff
@@ -7,11 +7,63 @@
...
 Supported platforms:
+- Added TCP, CUDA, and ROCM transports
 Features:
+- Added stream-based communication API to UCP
+- Added client/server based connection establishment
+- Support for hardware tag-matching offload for DC and accelerated transports
+- Multi-rail support for eager and rendezvous protocols
+- Added support for tag-matching communications with CUDA buffers
+- Added ucp_rkey_ptr() to obtain pointer for remote memory
+- Avoid progress overhead on unused transprots
```
Comment (miked-mellanox) [inline NEWS]: s/transprots/transports

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6144/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shamisp @bbenton pls take a look

**Comment by shamisp** [comment]
I would not be supper eager to merge it in since this is something we will
be updating over the release.

**Comment by yosefe** [comment]
the release is in a week so we need to have it

**Comment by shamisp** [comment]
you are optimist

**Comment by shamisp** [comment]
Let's add MOFED, rdma-core requirements.
Knem, xpmem requirements.
@MattBBaker similarly for Cray.

On what arch it was tested, for arm I would suggest to include specific uarch (tx1, tx2, etc.). What

File: NEWS [hunk 2]
```diff
@@ -7,11 +7,63 @@
 ##
 #
 
-## Next Release
+## 1.3.0 (February 15, 2018)
+
 Supported platforms:
+- Added TCP, CUDA, and ROCM transports
```
Comment (shamisp) [inline NEWS]: Move to a new line:
- Added support for GPU platforms: Nvidia CUDA and AMD ROCM software stacks

File: NEWS [hunk 3]
```diff
@@ -7,11 +7,63 @@
 ##
 #
 
-## Next Release
+## 1.3.0 (February 15, 2018)
+
 Supported platforms:
```
Comment (shamisp) [inline NEWS]: New platforms and interconnects

File: NEWS (hunk 2 above)
Comment (shamisp) [inline NEWS]: Added support for TCP

File: NEWS [hunk 4]
```diff
@@ -7,11 +7,63 @@
 ##
 #
 
-## Next Release
+## 1.3.0 (February 15, 2018)
```
Comment (shamisp) [inline NEWS]: Software stack requirements:
MOFED version
RDMA core version
CUDA version
ROCM version
XPMEM version
KNEM version

File: NEWS [hunk 5]
```diff
@@ -7,11 +7,63 @@
...
+
 Supported platforms:
+- Added TCP, CUDA, and ROCM transports
 Features:
+- Added stream-based communication API to UCP
+- Added client/server based connection establishment
+- Support for hardware tag-matching offload for DC and accelerated transports
```
Comment (shamisp) [inline NEWS]: for InfiniBad hardware tag-matching ...

File: NEWS [hunk 6]
```diff
@@ -7,11 +7,63 @@
...
 #
 
-## Next Release
+## 1.3.0 (February 15, 2018)
+
 Supported platforms:
+- Added TCP, CUDA, and ROCM transports
 Features:
+- Added stream-based communication API to UCP
+- Added client/server based connection establishment
```
Comment (shamisp) [inline NEWS]: Added API for ..

File: NEWS [hunk 7]
```diff
@@ -7,11 +7,63 @@
...
+- Support for hardware tag-matching offload for DC and accelerated transports
+- Multi-rail support for eager and rendezvous protocols
+- Added support for tag-matching communications with CUDA buffers
+- Added ucp_rkey_ptr() to obtain pointer for remote memory
```
Comment (shamisp) [inline NEWS]: for shared memory region

File: NEWS [hunk 8]
```diff
@@ -7,11 +7,63 @@
...
+- Avoid progress overhead on unused transports
+- Improved scalability of software tag-matching by using a hash table
+- Added transparent huge-pages allocator
+- Added non-blocking flush and disconnect for UCP
+- Support fixed-address memory allocation via ucp_mem_map()
+- Added ucp_tag_send_nbr() API to avoid send request allocation
+- Support global addressing in all IB transports
+- Add support for external epoll fd and edge-triggered events
+- Added registration cache for knem
+- Initial support for Java bindings
```
Comment (shamisp) [inline NEWS]: New JAVA API. Should go to the top

File: NEWS [hunk 9]
```diff
@@ -7,11 +7,63 @@
...
 Bugfixes:
+- Multiple bugfixes
```
Comment (shamisp) [inline NEWS]: (see GitHub )

File: NEWS (hunk 8 above)
Comment (yosefe) [reply NEWS]: in this release, it's pre-alpha

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6219/ for details (Mellanox internal link).

**Comment by bureddy** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6226/ for details (Mellanox internal link).

**Comment by bureddy** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6230/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shamisp can we merge this? we can add more to the NEWS later if needed

**Pull request merged**